awx/ui_next/src/screens/Job/JobOutput/JobOutput.jsx

```diff
@@ -63,6 +63,44 @@ const OutputFooter = styled.div`
   flex: 1;
 `;
 
+let ws;
+function connectJobSocket({ type, id }, onMessage) {
+  ws = new WebSocket(`wss://${window.location.host}/websocket/`);
+
+  ws.onopen = () => {
+    const xrftoken = `; ${document.cookie}`
+      .split('; csrftoken=')
+      .pop()
+      .split(';')
+      .shift();
+    const eventGroup = `${type}_events`;
+    ws.send(
+      JSON.stringify({
+        xrftoken,
+        groups: { jobs: ['summary', 'status_changed'], [eventGroup]: [id] },
+      })
+    );
+  };
+
+  ws.onmessage = e => {
+    onMessage(JSON.parse(e.data));
+  };
+
+  ws.onclose = e => {
+    // eslint-disable-next-line no-console
+    console.log('Socket closed. Reconnecting...', e);
```

Comment:
could we make this console.debug (or one of the other ones), sometimes these messages start infiltrating output you might want to get access to in the console when debugging

Reply:
good idea - done